pkg/cmd/server/admin/create_mastercerts.go

```diff
@@ -113,6 +114,8 @@ func NewCommandCreateMasterCerts(commandName string, fullName string, out io.Wri
 	flags.StringSliceVar(&options.Hostnames, "hostnames", options.Hostnames, "Every hostname or IP that server certs should be valid for (comma-delimited list)")
 	flags.BoolVar(&options.Overwrite, "overwrite", false, "Overwrite all existing cert/key/config files (WARNING: includes signer/CA)")
 
+	flags.IntVar(&options.ExpireDays, "expire-days", options.ExpireDays, "Validity of the certificates in days (defaults to 2 years, you should not change this lightly).")
```

Comment:
This currently show something like that with `--help`:
```
 --expire-days=0: Validity of the certificates in days (defaults to 2 years, you should not change this lightly).
```
You should make sure to set `options.ExpireDays` to 2 years a few lines before, where `CreateMasterCertsOptions` is initialized. The 2 years in the square might stay since the value won't be easily understandable, but should at least say that 2 years in no of days.